doc/Makefile.sphinx

```diff
@@ -61,23 +66,32 @@ check-sphinxbuild:
 	@echo "Confirming that sphinx-build is available..."
 	@if [ $(SPHINXTEST) -eq 1 ]; then echo $(SPHINXERROR); fi
 
-html: check-sphinxbuild source clean-build
+html-release: check-sphinxbuild source clean-build
 	@echo
 	@echo "Building HTML documentation"
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@chmod -R ugo+rX $(BUILDDIR)
+	mv $(BUILDDIR)/html .
 	@echo
-	@echo "Build finished. The HTML pages are in "'$(BUILDPATH)'"/html."
+	@echo "Build finished. The HTML pages are in "'$$CHPL_HOME'"/doc/html."
 
 html-dev:
 	@echo
 	@echo "Building HTML documentation with no prerequisites (dev testing)"
 	@echo "Warnings not treated as errors"
 	$(SPHINXBUILD) -b html -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SOURCEDIR) $(BUILDDIR)/html
 	@chmod -R ugo+rX $(BUILDDIR)
+	mv $(BUILDDIR)/html .
```

Comment:
This target does not depend on `clean-build` (intentionally, to save time on rebuilds). I think I need to add a manual `rm -rf html` here and change the `mv` to a `cp -r`.